global.pot

```diff
@@ -6,7 +6,7 @@ msgstr ""
 "PO-Revision-Date: 2018-09-26 18:26+1200\n"
 "Last-Translator: Pierre-Henry Soria <hi@ph7.me>\n"
 "Language-Team: ¡pH7! - Pierre-Henry Soria <hello@ph7cms.com>\n"
-"Language: en_US\n"
+"Language:  pt_Brasil  \ n"
```

Comment:
This is already done the the gettext Brazilian file https://github.com/pH7Software/pH7-Internationalization/blob/master/_protected/app/langs/pt_BR/LC_MESSAGES/global.po#L9

The one you changed is the POT file. POT file is basically the template file for PO files. The one where you start a new translation. So, the `global.pot` has to stay in English.

https://ph7cms.com/doc/en/how-to-translate-to-another-language